addons/es/verifactu/tax_test.go

```diff
@@ -85,4 +86,66 @@ func TestValidateTaxCombo(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("exempt with valid reason", func(t *testing.T) {
+		tc := &tax.Combo{
+			Category: tax.CategoryVAT,
+			Ext: tax.Extensions{
+				ExtKeyRegime: "01",
+				ExtKeyExempt: "E1",
+			},
+		}
+		err := validateTaxCombo(tc)
+		assert.NoError(t, err)
+	})
+
+	t.Run("excludes E2 exemption code with regime 01", func(t *testing.T) {
+		tc := &tax.Combo{
+			Category: tax.CategoryVAT,
+			Ext: tax.Extensions{
+				ExtKeyRegime: "01",
+				ExtKeyExempt: "E2",
+			},
+		}
+		err := validateTaxCombo(tc)
+		assert.Error(t, err)
+		assert.Contains(t, err.Error(), "E2")
```

Comment:
Two birds, one stone:
```suggestion
		assert.ErrorContains(t, err, "E2")
```